legacy/cli/audit.go

```diff
@@ -42,13 +43,17 @@ func RunAuditCmd(opts *AuditOptions) error {
 		return errors.Wrap(err, "validating audit options")
 	}
 
+	// Allow persistent logging when --verbose flag is present.
+	if opts.Verbose {
```

Comment:
Why not do something like

```
logclient.IsVerbose = opts.Verbose
```

instead of the `if` statement here to initialize the `IsVerbose` global variable (we don't have one yet, but we can use one) in the logclient? Seems simpler than calling a function (`EnablePersistentLogging`) in order to modify a member field in the `PersistentAuditInfo` global struct.